cmssw-tool-conf.spec

```diff
@@ -68,6 +68,7 @@ Requires: python-toolfile
 Requires: python3-toolfile
 Requires: root-toolfile
 Requires: sherpa-toolfile
+Requires: hwloc-toolfile
```

Comment:
@fwyzard , you need to add all the newly added toolfiles dependency here (specialy if libs/tools are needed at runtuime)

Reply:
Are they needed also if CMSSW does not need to use them ?
For example, CMSSW uses OpenMPI, OpenMPI uses UCX, and UCX (conditionally) uses GDRCopy.
But I don't expect any CMSSW code to directly use UCX or GDRCopy.

Are the toolfiles still needed ?

Reply:
SCRAM needs tool files to properly  set env (e.g. LD_LIBRARY_PATH and PATH), so if libs and executables are needed at runtime (e.g. if ucx libs are linked to openmpi libs)  then we need to add ucx-toolfile dependency in cmssw-tool-conf.